src/components/Text/Text.js

```diff
@@ -8,7 +8,7 @@ const Text = styled(({
   ...props
 }) => React.createElement(component, props, children))`
   ${(p) => p.color ? `color: ${p.theme.color[p.color]};` : ''}
-  ${(p) => p.size ? `font-size: ${p.theme.fontSize[p.size]};` : ''}
+  ${(p) => p.size ? `font-size: ${p.theme.fontSize[p.size]}px;` : ''}
```

Comment:
Won't this default to `px` anyway?

Reply:
I wasn't sure. Hence put it here. Will take it off.